models/rmResponse.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+type RmResponse struct {
+	CaseID          *string `json:"caseId"` // Why a reference type?
```

Comment:
Did we answer the question "Why a reference type?" yet?

Reply:
no we didn't, and i think the answer is "it doesn't need to be" so will fix 👍

Reply:
@NickGrantONS this is fixed now 👍